spec/example_app/app/assets/stylesheets/administrate/customize.scss

```diff
@@ -0,0 +1,4 @@
+.custom-administrate-css-hidden {
```

Comment:
The name of this file should begin with an underscore, to align with standards: `_customize.scss`